garak/_config.py

```diff
@@ -20,6 +20,8 @@
     xdg_config_home,
     xdg_data_home,
 )
+from garak.translator import SimpleTranslator, EncodingTranslator, GoodsideTranslator, DanTranslator, ReverseTranslator
+from garak.translator import LocalDanTranslator, LocalTranslator, LocalEncodingTranslator, LocalGoodsideTranslator, LocalReverseTranslator
```

Comment:
using the `from` pattern means this must be explicitly maintained as translators come & go. recommend using `import garak.translator` and being more verbose later

Reply:
OK. I'll refactor it.

> also: do all these classes need to be imported any time garak._config is imported? (which is, a lot)

No, I don't. It can import it when translation functions need it